config/initializers/omniauth.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2,
+  ENV["GOOGLE_CLIENT_ID"],
+  ENV["GOOGLE_CLIENT_SECRET"],
+  {name: 'google_login', approval_prompt:''}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.